hack/kind-up.sh

```diff
@@ -381,16 +381,21 @@ EOF"
   docker exec "$node" sh -c "systemctl enable gardener-local-kind-add-hosts.service"
 done
 
+local_address_operator="172.18.255.3"
```

Comment:
```suggestion
virtual_garden_ip="172.18.255.3"
```

or

```suggestion
local_address_virtual_garden="172.18.255.3"
```

Maybe we can even rename `garden_cluster_ip` to `runtime_garden_ip`?

Reply:
I renamed the `local_address_operator` variable to `local_address_virtual_garden`, but kept `garden_cluster_ip`.
In the non-operator setup, we use `garden.local.gardener.cloud` (pointing to `$garden_cluster_ip`) to connect to the garden cluster (which is the first kind cluster). In the operator setup, this cluster is the "garden runtime cluster" but here the hostname is irrelevant/not used.